src/components/PostList/PostList.jsx

```diff
@@ -1 +1,14 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => {
+  return (
+    <div className="PostList">
+      {posts.map(post => (
+        <PostInfo
+          key={post.id}
+          post={post}
+        />
+      ))}
+    </div>
+  );
+};
```

Comment:
since the function only returns value without any logic - we can make it return right away
```suggestion
export const PostList = ({ posts }) => (
  <div className="PostList">
    {posts.map(post => (
      <PostInfo
        key={post.id}
        post={post}
      />
    ))}
  </div>
);
```